multiqc/modules/hicpro/hicpro.py

```diff
@@ -0,0 +1,386 @@
+             'shared_key' : 'read_count',
+             'hidden': True
+         }
+
+         headers['percent_mapped_R2'] = {
+             'title': '% Aligned [R2]',
+             'description': 'Percentage of aligned reads [R2] (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': True
+         }
+
+         headers['paired_reads'] = {
+             'title': 'Reported Read Pairs',
+             'description' : 'Total number of reported read pairs ({})'.format(config.read_count_desc),
+             'min' : '0',
+             'scale' : 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key' : 'read_count',
+             'hidden': True
+         }
+
+         headers['percent_paired_reads'] = {
+             'title': '% Reported Read Pairs',
+             'description': 'Percentage of paired reads (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': False
+         }
+
+         headers['valid_interaction'] = {
+             'title': '{} Valid Pairs'.format(config.read_count_prefix),
+             'description': 'Number of valid pairs ({})'.format(config.read_count_desc),
+             'min': 0,
+             'scale': 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key': 'read_count',
+             'hidden': True
+         }
+         
+         headers['percent_valid'] = {
+             'title': '% Valid Pairs',
+             'description': 'Percentage of valid pairs over reported ones (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': True
+         }
+         
+         headers['valid_interaction_rmdup'] = {
+             'title': '{} Valid Pairs Unique'.format(config.read_count_prefix),
+             'description': 'Number of valid pairs after duplicates removal ({})'.format(config.read_count_desc),
+             'min': 0,
+             'scale': 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key': 'read_count',
+             'hidden': True
+         }
+         
+         headers['percent_valid_interaction_rmdup'] = {
+             'title': '% Valid Pairs Unique',
+             'description': 'Percent of valid pairs over reported ones after duplicates removal (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': False
+         }
+     
+         self.general_stats_addcols(self.hicpro_data, headers, 'HiC-Pro')
+
+
+    def hicpro_mapping_chart (self):
+        """ Generate the HiC-Pro Aligned reads plot """
+
+        # Specify the order of the different possible categories
+        keys = [OrderedDict(), OrderedDict()]
```

Comment:
Minor thing, but this can be simplified - just create one `OrderedDict()` and use it twice:

```python
return bargraph.plot(data, [keys, keys], config)
```

Reply:
indeed !